src/state/ppangoChef/hooks.ts

```diff
@@ -349,3 +358,64 @@ export function usePangoChefInfos() {
     pairs,
   ]);
 }
+
+export function useUserAPR(stakingInfo?: PangoChefInfo) {
```

Comment:
lets rename this hook to indicate its PangoChef APR something like `useUserPangoChefAPR`